src/main/resources/default/templates/biz/login.html.pasta

```diff
@@ -100,13 +104,28 @@
         sirius.ready(function () {
             const _userField = document.getElementById('user');
             const _passwordField = document.getElementById('password');
+            const _togglePasswordButton = document.getElementById('passwordToggleButton');
 
             if (!sirius.isEmpty(_userField.value)) {
                 _passwordField.focus();
             } else {
                 _userField.focus();
             }
 
+            _togglePasswordButton.addEventListener('click', function () {
+                const _togglePasswordIcon = document.getElementById('passwordToggleIcon');
+
+                if (_passwordField.getAttribute('type') === 'password') {
+                    _passwordField.setAttribute('type', 'text');
+                    _togglePasswordIcon.classList.remove('fa-eye');
+                    _togglePasswordIcon.classList.add('fa-eye-slash');
```

Comment:
Ich frage mich, ob das hier immer funktioniert. Gab es nicht auch einen Rendering Mode von FA, wo das `<i>` ersetzt wird durch ein eingebettetes `<svg>`, wo man dann nicht einfach die `fa-` Klasse austauschen kann? Hast Du das in mehreren Browsern probiert? (Alternativ könnte man beide Icons in den DOM-Tree rendern und mittels `d-none` an einem umfassenden `<div>` die Sichtbarkeit hin und her schalten.)

Reply:
...der Modus wird hier verwendet, siehe commit-message. Daher erst im onclick das Element.
Getestet in Chrome, FIrefox, Safari

Reply:
Bitte auch als Kommentar im Code